yt_dlp/extractor/ard.py

```diff
@@ -383,7 +403,9 @@ def _real_extract(self, url):
         media_data = traverse_obj(player_data, ('mediaCollection', 'embedded', {dict}))
 
         if player_data.get('blockedByFsk'):
-            self.raise_no_formats('This video is only available after 22:00', expected=True)
+            self.raise_no_formats(
+                'This video is only available after 22:00, '
+                'try passing cookies to enable age verification ', expected=True)
```

Comment:
```suggestion
            self.raise_login_required(
                'This video is only available after 22:00. Try passing cookies to enable age verification')
```

Reply:
removing the `expected=True` is intentional here?

Reply:
yeah, `raise_login_required` is always expected

Reply:
did not even notice that change 😄  manually applied it now since the message is different now